docs/copied-from-beats/docs/loggingconfig.asciidoc

```diff
@@ -235,6 +235,12 @@ true.
 
 When true, logs messages in JSON format. The default is false.
 
+[float]
+==== `logging.ecs`
+
+When true, logs messages with minimal required Elastic Common Schema (ECS)
+information.
+
```

Comment:
https://github.com/elastic/apm-server/pull/3749 was merged into `7.x`, not `master`. Is this ok to have in `master`?

Reply:
I'm pretty certain it is. @simitt?

Reply:
yes it is, sorry I forgot to follow up with it in master.